google-cloud-firestore/src/main/java/com/google/cloud/firestore/DocumentSnapshot.java

```diff
@@ -34,9 +34,13 @@
 
 /**
  * A DocumentSnapshot contains data read from a document in a Firestore database. The data can be
- * extracted with the getData or get methods.
+ * extracted with the {@code getData()} or {@code get()} methods.
```

Comment:
Why not use links for references to methods? e.g. `{@link #getData()}`

Reply:
Done